backend/src/main/java/com/woowacourse/kkogkkog/presentation/MemberController.java

```diff
@@ -43,8 +45,15 @@ public ResponseEntity<Void> updateMe(@LoginMember Long id,
         return ResponseEntity.ok().build();
     }
 
-    @GetMapping("/me/history")
+    @GetMapping("/me/histories")
     public ResponseEntity<MemberHistoriesResponse> showHistory(@LoginMember Long id) {
         return ResponseEntity.ok(new MemberHistoriesResponse(memberService.findHistoryById(id)));
     }
+
+    @PatchMapping("/me/histories/{historyId}")
+    public ResponseEntity<Void> updateMemberHistory(@PathVariable Long historyId) {
+        memberService.updateMemberHistory(historyId);
```

Comment:
patch가 거슬리긴 하는데, 현재 상황에서 put보다는 어울리는 것 같아서 우선 이대로 진행해도 될 것 같습니다.
```suggestion
    @PatchMapping("/me/histories/{historyId}")
    public ResponseEntity<Void> updateIsReadMemberHistory(@PathVariable Long historyId) {
        memberService.updateMemberHistory(historyId);
```